.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -21,4 +21,7 @@
     <!-- Generated code should not be subjected to checkstyle. -->
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_testSrc[/\\]" checks="." />
+
+    <!-- Exception for SunPkcs1Reader -->
+    <suppress files="SunPkcs1Reader.java" checks="IllegalImport" />
```

Comment:
Opened https://github.com/palantir/gradle-baseline/issues/51 to discuss and submitted https://github.com/palantir/gradle-baseline/pull/52 as a PR for a way that this could be done that I think makes sense.